app/components/pipeline/workflow/styles.scss

```
@@ -26,6 +26,7 @@
     #workflow-display-container {
       grid-area: workflow;
       overflow: auto;
+      padding-right: max(1rem, 0.8vw);
```

```suggestion
      padding-right: 1rem;
```
You'll want to keep this as 1rem to keep things lined up along the right side of the window